ts/src/woo.ts

```diff
@@ -837,7 +836,19 @@ export default class woo extends Exchange {
         if (amount !== undefined) {
             request['quantity'] = this.amountToPrecision (symbol, amount);
         }
-        const response = await this.v3PrivatePutOrderOid (this.extend (request, params));
+        const clientOrderIdUnified = this.safeString2 (params, 'clOrdID', 'clientOrderId');
+        const clientOrderIdExchangeSpecific = this.safeString (params, 'client_order_id', clientOrderIdUnified);
+        const isByClientOrder = clientOrderIdExchangeSpecific !== undefined;
+        let method = undefined;
+        if (isByClientOrder) {
+            method = 'v3PrivatePutOrderClientClientOrderId';
```

Comment:
@sc0Vu next time let's avoid `method = 'v3PrivatePutOrderClientClientOrderId'` we now have the implicit API methods so we should use them to avoid typos in the endpoint name

Reply:
@carlosmiei do you mean use `response = await this. v3PrivatePutOrderClientClientOrderId (...)` directly?

Reply:
@sc0Vu yes (if possible)